plugins.js

```diff
@@ -0,0 +1,51 @@
+module.exports = [
+  {
+    name: 'postcss-simple-vars',
+  },
+  {
+    name: 'postcss-advanced-variables',
+    config: { disable: ['@import'] },
+    url: 'https://github.com/jonathantneal/postcss-advanced-variables',
```

Comment:
I take it this plugin is not on https://github.com/himynameisdave/postcss-plugins? If so, we should probably just add it to the list there.

This isn't a blocker for merging; just a note that we should do this sometime.

Reply:
yes indeed. If @jonathanneal does not want it to be on postcss.parts for some reason, we can always put the data just here as well. I let him decide which name/description he wants for this

Reply:
oops, sorry for the bother. I mean @jonathantneal

Reply:
I would want it there. I just forgot to add plugins after a certain time.

Reply:
PR done https://github.com/himynameisdave/postcss-plugins/pull/231